### PR TITLE
Add implementation

### PR DESCRIPTION
**Pull request by YevheniiSemendiak**
@anayden, pls check if I haven't missed or mistyped something.
Thank you! <3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+```yaml
+args:
+    POSTGRES_INITDB_ARGS: "--data-checksums"
+```
+
+
+# Known issues
+### `chmod: changing permissions of '...': Operation not permitted`
+This might happen if the platform `storage:` backend does not support such an operation (happened with Azure Files and CIFS).
+A work-around for this is to use a platform `disk:` to host the SQLite data.
```
Comment (anayden) [inline README.md]: I guess SQLite is a typo here :)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+Resource preset to use when running the Postgres job. `""` by default (i.e., the first preset specified in the `neuro config show` list will be used).
+
+#### Example
+```yaml
+args:
+    preset: cpu-small
+```
+
+
+### `http_auth`
```
Comment (anayden) [inline README.md]: How would one use http auth? As far as I know, Postgres doesn't provide web ui

File: action.yaml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    default: 14
+    type: str
+    descr: |
+      PostgreSQL server version, used as an image tag.
+      Checkout list of available tags at https://hub.docker.com/_/postgres?tab=tags. 
+  db_user:
+    default: postgres
+    descr: |
+      Password to the DB, will be used as Job ENV variable.
+      Do not use the default value, instead consider Neu.ro secret: usage.
```
Comment (anayden) [inline action.yaml]: If we say "do not use default", let's remove the default.
Same for password

File: action.yaml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  db_password:
+    default: password
+    descr: |
+      Password to the DB, will be used as Job ENV variable.
+      Do not use the default value, instead consider Neu.ro secret: usage.
+  postgres_initdb_args:
+    default: ""
+    type: str
+    descr: |
+      This optional environment variable can be used to send arguments to `postgres initdb`.
```
Comment (anayden) [inline action.yaml]: This is an argument, not environment variable

**Review by anayden — CHANGES_REQUESTED** [review]
LGTM overall, but feels like some small improvements are needed

File: README.md (hunk 2 above)
Comment (YevheniiSemendiak) [reply README.md]: Hmm, googled it a bit, but you're right, HTTP API should be installed separately.
I've set it to true by default in 942d5b9 and removed it from the readme in 2c9dae0

File: action.yaml (hunk 3 above)
Comment (YevheniiSemendiak) [reply action.yaml]: completely agree! thank you for a good advice, applied in 2c9dae0

File: action.yaml (hunk 4 above)
Comment (YevheniiSemendiak) [reply action.yaml]: correct, fixed in c84d841

File: README.md (hunk 1 above)
Comment (YevheniiSemendiak) [reply README.md]: Yeah, thank you! c84d84196f85

**Review by anayden — APPROVED** [review]
Sorry, apparently I missed the PR in the past. All good!

**Pull request merged**